src/StrawberryShake/Tooling/src/Configuration/VisibilityOverride.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace StrawberryShake.Tools.Configuration
+{
+    public enum VisibilityOverride
+    {
+        /// <summary>
+        /// Use the default visibility.
+        /// </summary>
+        Default,
```

Comment:
The default can go since the default on the class should be just our default.

Reply:
Done.